### PR TITLE
Add subcellular application use case

### PR DESCRIPTION
**Pull request by pgetta**

### CODE REVIEW

File: source/online_usecases/subcellular_level/subcellular_app/subcellular_app.rst [hunk 1]
```diff
@@ -0,0 +1,265 @@
...
+which provides spatial stochastic and deterministic solvers for simulation
+of reactions and diffusion on tetrahedral meshes.
+
+Application as well provides a number of facilities for a visualization
+of models geometry and results of simulations.
+
+The molecular repository is a publicly available database of biological
+information relevant for brain molecular network modeling.
+
+It accomodates several types of biological information which are not available
```
Comment (alex4200) [inline source/online_usecases/subcellular_level/subcellular_app/subcellular_app.rst]: acommodates (one 'm' missing)

File: source/online_usecases/subcellular_level/subcellular_app/subcellular_app.rst [hunk 2]
```diff
@@ -0,0 +1,265 @@
...
+    * To load a public model click an arrow to expand a list of public models
+      in `Subcellular models repository`, choose one, then click `Load`
+    * To import `BNGL`_ or `SBML`_ (`Atomizer`_ is used for latter) model click
+      `Import` in `Model` menu
+    * To create a new model specify all model parameters and reaction equations
+      sequentially opening and filling tables in the `Model` menu (upper left
+      of the UI). Each table corresponds to a particular section of the
+      BioNetGen file of the model (with except for `Diffusions` and `Geometry`
+      section). All expressions should comply with BioNetGen language.
+#.  To add diffusion (needed if using STEPS) navigate to corresponding section
```
Comment (alex4200) [inline source/online_usecases/subcellular_level/subcellular_app/subcellular_app.rst]: I would exchange 'needed' by 'required' to be consistent with the next sentence

File: source/online_usecases/subcellular_level/subcellular_app/subcellular_app.rst (hunk 1 above)
Comment (pgetta) [reply source/online_usecases/subcellular_level/subcellular_app/subcellular_app.rst]: Fixed

File: source/online_usecases/subcellular_level/subcellular_app/subcellular_app.rst (hunk 2 above)
Comment (pgetta) [reply source/online_usecases/subcellular_level/subcellular_app/subcellular_app.rst]: Changed

**Comment by lbologna** [comment]
Hello,

@alex4200 @pgetta I just added the new page to dev (with some minor changes to the text):
https://lbologna.github.io/hbp-sp6-guidebook/online_usecases/subcellular_level/subcellular_app/subcellular_app.html
Please let me know if this is fine and we can put it to prod.
Thanks

**Comment by alex4200** [comment]
@lbologna To me it looks ok. You can move it to production.

**Comment by pgetta** [comment]
Thank you, @lbologna! Looks good to me.

**Comment by lbologna** [comment]
Thanks!
Merged to Prod.
@pgetta @alex4200

**Pull request merged**